src/main/webapp/script.js

```diff
@@ -103,3 +103,33 @@ function getTimeAgo(timeInMilli) {
   }
   return Math.floor(months / 12) + " years ago";
 }
+
+function getLyrics() {
+  document.getElementById('error-lyric').innerHTML = '';
+  // Gets value from inputs and sets them
+  var artist = document.getElementById('artist').value;
+  var song = document.getElementById('song').value;
+  var request = new XMLHttpRequest();
+  var obj;
+  // Calls lyrics.ovh API
+  request.open('GET', 'https://api.lyrics.ovh/v1/' + artist + '/' + song);
+  // Returns lyrics in JSON format
+  request.onreadystatechange = function () {
+    if (this.readyState === 4) {
+      var lyricsJSON = this.responseText;
+      // API will return two key values, error (if not found), 
+      // or lyrics (if found, but could possibly return empty string)
+      obj = JSON.parse(lyricsJSON);
+      if (obj.lyrics && obj.lyrics != '' && obj.lyrics != null) {
+        // Call function that does sentiment analysis on obj.lyrics string
+        document.getElementById("form").submit();
+      } else {
+        document.getElementById('error-lyric').innerHTML = 'Could not find song.' + 
+          ' Check spelling, try to submit again, or choose another song.';
```

Comment:
Quick fix: could you delete the space before "Check"?

Reply:
The space before "Check", is used to properly space the sentence.
With space: Could not find song. Check spelling, try to submit again, or choose another song.
Without space: Could not find song.Check spelling, try to submit again, or choose another song.